flink/v1.17/flink/src/test/java/org/apache/iceberg/flink/source/TestIcebergSourceFailover.java

```diff
@@ -98,9 +98,9 @@ protected List<Record> generateRecords(int numRecords, long seed) {
     return RandomGenericData.generate(schema(), numRecords, seed);
   }
 
-  protected void assertRecords(
-      Table table, List<Record> expectedRecords, Duration interval, int maxCount) throws Exception {
-    SimpleDataUtil.assertTableRecords(table, expectedRecords, interval, maxCount);
+  protected void assertRecords(Table table, List<Record> expectedRecords, Duration interval)
```

Comment:
do we need this method? can't we replace with 
`SimpleDataUtil.assertTableRecords(table, expectedRecords, interval);` ?

Reply:
that's a valid point, I'm not sure why this method existed in the first place. I've removed it